### PR TITLE
adding node scale

### PR DESCRIPTION
**Pull request by cixzhang**
Adding the ability to scale nodes as a Grapher transform. This should cover any cases that need the node scaling but lets the user specify how it's handled.

### CODE REVIEW

File: modules/grapher.js [hunk 1]
```diff
@@ -366,6 +367,20 @@ Grapher.prototype.transform = function (transform) {
...
 /**
+  * grapher.nodeScale
+  * ------------------
+  *
+  * Set the nodeScale. The displayed radius of the node will be the nodeScale multiplied by the node radius.
+  * If no arguments are passed in, returns the current scale.
+  */
+Grapher.prototype.nodeScale = function (scale) {
+  if (u.isUndefined(scale)) return this.props.nodeScale;
+  if (u.isNumber(scale)) this.props.nodeScale = scale;
```
Comment (DianaSuvorova) [inline modules/grapher.js]: Do you want to add && !u.isNaN(scale) ?

Comment (cixzhang) [inline modules/grapher.js]: I'm treating NaNs as if they were undefined. I hope that makes sense!

Comment (DianaSuvorova) [inline modules/grapher.js]: typeof NaN === 'number' if we want to double check that scale is not a NaN, there needs to be additional check.  But this is not a big deal anyways.

**Comment by cixzhang** [comment]
What I mean is passing a NaN to this function will result in the same behavior as passing undefined. You can see this in the changes made here https://github.com/ayasdi/grapher/commit/2f4845ed93457865cfc1b787cbc570cfec9f8ed6

**Pull request merged**